packages/enhanced/package.json

```diff
@@ -5,9 +5,6 @@
   "type": "commonjs",
   "main": "src/index.js",
   "types": "src/index.d.ts",
-  "publishConfig": {
```

Comment:
heads up: without publishConfig here, CI will not publish to right registry.

Reply:
According to the documentation the [publishConfig](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#publishconfig) option extends the [config type](https://docs.npmjs.com/cli/v10/using-npm/config) values, and the [registry](https://docs.npmjs.com/cli/v10/using-npm/config#registry) property uses npm by default, but defining the registry option using publishConfig is something hard to override during publish, even specifying different values on the CLI. So I think it would be better if we find a way to enforce the CI to use the right registry by passing it as an argument or making it use the default npm value (https://registry.npmjs.org/)

Reply:
i think it may be becuase something in here still points to or uses yarn, CI usually tries to hit yarn registry. Ill dig around in here and see if i can find where the default is getting set